app/models/expense.rb

```diff
@@ -0,0 +1,8 @@
+class Expense < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :expense_categories, dependent: :destroy
+  has_many :categories, through: :expense_categories, dependent: :destroy
```

Comment:
[OPTIONAL] I would recommend removing line 3 since the Expense has many categories through expanse_categories.